hack/verify-upstream-featuregates.sh

```
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+source "$(dirname "${BASH_SOURCE}")/lib/init.sh"
+
+VERIFY_DIR=$(mktemp -d -t upstream-featuregates-verify-XXXXXX)
+
+go run --mod=vendor -trimpath github.com/openshift/api/payload-command/cmd/write-upstream-featuregates --output-directory=${VERIFY_DIR} --output-filename=zz_generated.upstreamfeaturegates.go
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)